typescript/radix-sdk/src/tests/warp.e2e-test.ts

```
@@ -0,0 +1,791 @@
+        }
+
+        // Verify router enrollment, ISM, AND ownership transfer succeeded
+        const reader = artifactManager.createReader(type);
+        const readToken = await reader.read(deployedToken.deployed.address);
+
+        expect(readToken.config.remoteRouters[DOMAIN_1].address).to.equal(
+          '0xc2c6885c3c9e16064d86ce46b7a1ac57888a1e60b2ce88d2504347d3418399c4',
+        );
+        expect(
+          eqAddressRadix(
+            readToken.config.interchainSecurityModule?.deployed.address!,
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)